### PR TITLE
update get-started doc

### PR DESCRIPTION
**Pull request by DanielZhangQD**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/get-started.md [hunk 1]
```diff
@@ -618,7 +619,15 @@ mysql> select * from information_schema.cluster_info\G
 kubectl port-forward -n tidb-cluster svc/basic-grafana 3000 > pf3000.out &
 ```
 
-Grafana 面板可在 kubectl 所运行的主机上通过 <http://localhost:3000> 访问。默认用户名和密码都是 "admin" 。请注意，如果你是非本机（比如 Docker 容器或远程服务器）上运行 `kubectl port-forward`，将无法在本地浏览器里通过 `localhost:3000` 访问。
+Grafana 面板可在 kubectl 所运行的主机上通过 <http://localhost:3000> 访问。默认用户名和密码都是 "admin" 。
+
+请注意，如果你是非本机（比如 Docker 容器或远程服务器）上运行 `kubectl port-forward`，将无法在本地浏览器里通过 `localhost:3000` 访问，可以通过下面命令监听所有地址：
+
+``` bash
```
Comment (ran-huang) [inline zh/get-started.md:626]:
```suggestion
```bash
```

File: zh/get-started.md [hunk 2]
```diff
@@ -618,7 +619,15 @@ mysql> select * from information_schema.cluster_info\G
...
+kubectl port-forward --address 0.0.0.0 -n tidb-cluster svc/basic-grafana 3000 > pf3000.out &
+```
+
+然后通过 <http://远程服务器 IP:3000> 访问 Grafana。
```
Comment (ran-huang) [inline zh/get-started.md:630]:
```suggestion
然后通过 <http://${远程服务器IP}:3000> 访问 Grafana。
```

File: zh/get-started.md [hunk 3]
```diff
@@ -670,20 +679,22 @@ basic-tikv-0                      1/1     Running       0          4m13s
...
 ### 转发 TiDB 服务端口
 
-当所有 Pods 都重启后，将看到版本号已更改。需要注意的是，由于相关 Pods 已被销毁重建，这里需要重新设置端口转发。如果 `kubeclt port-forward` 进程仍然在运行，请结束进程后再转发端口。
+当所有 Pods 都重启后，将看到版本号已更改。需要注意的是，由于相关 Pods 已被销毁重建，这里需要重新设置端口转发。
 
 {{< copyable "shell-regular" >}}
 
 ```
-kubectl port-forward -n tidb-cluster svc/basic-tidb 4000 > pf4000.out &
+kubectl port-forward -n tidb-cluster svc/basic-tidb 24000:4000 > pf24000.out &
```
Comment (ran-huang) [inline zh/get-started.md:687]: 是 24000 不是 14000 吗？

**Review by ran-huang — COMMENTED** [review]
Rest LGTM

File: zh/get-started.md (hunk 3 above)
Comment (DanielZhangQD) [reply zh/get-started.md:687]: 14000 上面已经被占用了，不会自动退出

File: zh/get-started.md [hunk 4]
```diff
@@ -710,6 +721,16 @@ Check Table Before Drop: false
 
 完成测试后，你可能希望销毁 TiDB 集群和 Kubernetes 集群。
 
+#### 停止 `kubectl` 的端口转发
```
Comment (DanielZhangQD) [inline zh/get-started.md:724]: 需要先停止转发，否则进行后面操作时终端会出现报错

**Comment by DanielZhangQD** [comment]
@ran-huang PTAL again.

**Review by ran-huang — APPROVED** [review]

**Review by csuzhangxc — APPROVED** [review]

**Pull request merged**